packages/fxa-settings/src/pages/ConfirmSignin/index.tsx

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { usePageViewEvent } from '../../lib/metrics';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { RouteComponentProps } from '@reach/router';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+
+export type ConfirmSigninProps = {
+  canGoBack?: boolean;
+  email: string;
+  goBackCallback?: Function;
+  isOpenWebmailButtonVisible: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+const ConfirmSignin = ({
+  canGoBack,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible,
+}: RouteComponentProps & ConfirmSigninProps) => {
+  usePageViewEvent('confirm-signin', {
+    entrypoint_variation: 'react',
+  });
+
+  // TO-DO: Replace utility that gets these values by matching the email provider via regex.
+  const getWebmailValues = (email: string) => {
+    return {
+      buttonText: 'Open Gmail',
+      link: `https://mail.google.com/mail/u/?authuser=${email}`,
+    };
+  };
+
+  const webmailValues = getWebmailValues(email);
+
+  const resendEmail = () => {
+    // TO-DO: resend email to user.
+  };
+
+  /*
+    TO-DO:
+        - Fix up the alert bar for any errors (such as from resending the email) or success.
+        - wire up the functionality to resend an email
+  */
+
+  return (
+    <>
+      <div className="mb-4">
+        <FtlMsg id="confirm-signin-header">
+          <h1 className="card-header">Confirm this sign-in</h1>
+        </FtlMsg>
+      </div>
+      <section>
+        <div className="flex justify-center">
+          <Mail role="img" />
+        </div>
+        <FtlMsg id="confirm-signin-message">
+          <p className="my-4 text-sm">
+            {`Check your email for the sign-in confirmation link sent to ${email}`}
+          </p>
+        </FtlMsg>
+        {isOpenWebmailButtonVisible && (
+          <div>
+            <LinkExternal
+              href={webmailValues.link}
+              className="link-blue text-sm"
+            >
+              {webmailValues.buttonText}
+            </LinkExternal>
+          </div>
+        )}
+        <>
+          {canGoBack && goBackCallback ? (
+            <div className="flex justify-around">
```

Comment:
`justify-around` adds some padding on the outer sides. Maybe `justify-between` would work here? I'm still deciding how to handle this styling for other components that have side-by-side links that might not render well with long localized text, so take this suggestion with a grain of salt.

Reply:
Hmmmm -- I see what you mean about long text. I'll stack them vertically just to be safe (but am leaving a note here so that we can remember and change it if needed!)